conf/messages

```diff
@@ -818,10 +821,10 @@ case.v2.liability.c592.date_for_repayment=Date for repayment
 
 case.v2.liability.c592.details_edit.information=Information from the C592
 case.v2.liability.c592.details_edit.entry_number=Entry number
-case.v2.liability.c592.details_edit.date_p=Entry date
+case.v2.liability.c592.details_edit.date_p=Liability entry date
 case.v2.liability.c592.details_edit.date_hint=Date recorded on the C592 by compliance or port officer, for example, 27 3 2015.
 case.v2.liability.c592.details_edit.bti_reference=BTI reference
-case.v2.liability.c592.details_edit.bti_reference_error=Enter a correct BTI reference number
+case.v2.liability.c592.details_edit.bti_reference_error=Enter a BTI/ ATaR case reference
```

Comment:
Really small thing, the spacing on BTI/ATaR is not the same between the two examples here